staging/src/k8s.io/apiserver/pkg/storage/storagebackend/factory/etcd3.go

```diff
@@ -218,13 +229,19 @@ func newETCD3Storage(c storagebackend.Config) (storage.Interface, DestroyFunc, e
 		return nil, nil, err
 	}
 
+	stopDbMetricMonitor, err := startDbMetricMonitorPerEndpoint(client, c.DbMetricPollInterval)
```

Comment:
DB. But shouldn't this name be stopDBSizeMonitor? Are we going to measure more than just the size?

Reply:
Renamed to stopDBSizeMonitor.